src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -1066,4 +1274,61 @@ function buildDotNetListItem(item: ListItem): DotNetListItem | null {
         children: children,
         actionSections: item.actionsSections
     } as DotNetListItem;
+}
+
+
+export function addReactiveWatcher(targetId: string, targetName: string, watchExpression: string, once: boolean, 
+                                   initial: boolean) : any {
+    let target = arcGisObjectRefs[targetId];
+    let dotNetRef = dotNetRefs[targetId];
+    console.log(`Adding watch: "${watchExpression}"`);
```

Comment:
if we make these `debug` will they go away in the compiled JS? Even if not, that might be nice to do at the debug level to reduce the amount of logging that is visible by default (log).

Reply:
I didn't even know that was a thing in JavaScript! Agreed, I'll play around with it.